engine/src/org/pentaho/di/trans/TransMeta.java

```diff
@@ -282,6 +282,9 @@
   /** The log channel interface. */
   protected LogChannelInterface log;
 
+  protected byte[] key;
```

Comment:
Let's use a more descriptive name in the code and the XML so people know what kind of key they are looking at.